app/src/main/java/com/jerboa/ui/components/common/Dialogs.kt

```diff
@@ -60,6 +55,8 @@ val DONATION_MARKDOWN = """
 """.trimIndent()
 
 val topSortTypes = SortType.values().filter { it.name.startsWith("Top") }
+val getSupportedSortTypes = { siteVersion: String -> SortType.values().filter { compareVersions(siteVersion, it.version) >= 0 } }
+val getSupportedCommentSortTypes = { siteVersion: String -> CommentSortType.values().filter { compareVersions(siteVersion, it.version) >= 0 } }
```

Comment:
Could this be moved to the companion object of the these enums. We probably want to reuse these on different locations